### PR TITLE
Added a bunch of governance documents relevant to the PSO application

### PR DESCRIPTION
**Pull request by rbeyer**

### CODE REVIEW

File: Contributing.md
Comment (michaelaye) [inline Contributing.md:21]: I think we need to be more clear here. We will have more than one repository? Unless we want to write PlanetaryPy GH Org? Not sure...

File: Charter.md
Comment (michaelaye) [inline Charter.md:58]: Do we have/need a general comment somewhere about lazy consent?

File: Charter.md
Comment (michaelaye) [inline Charter.md:76]: How about adding: Participation in the TC is **not** a requirement for becoming a Collaborator. Or is that self-understood by not mentioning it?

File: Contributing.md
Comment (michaelaye) [inline Contributing.md:43]: same as above, what `repository` is meant here? The whole org? Or the core package?

File: Contributing.md
Comment (michaelaye) [inline Contributing.md:73]: Even the PR author? I though we excluded that in PSO?

File: Contributing.md
Comment (michaelaye) [inline Contributing.md:99]: same as above, repositories?

File: PSO-Application.md
Comment (michaelaye) [inline PSO-Application.md:40]: is that sentence's grammar complete and okay? Feels broken off?

File: PSO-Application.md
Comment (michaelaye) [inline PSO-Application.md:117]: are u sure you want to store members in this document? or better link to TC members?

**Review by michaelaye — CHANGES_REQUESTED** [review]

File: Contributing.md
Comment (rbeyer) [reply Contributing.md:21]: So this is the `Contributing` document for the TC repo. The planetarypy repo and any affiliated package repos will also need their own `Contributing` documents that will probably be derived from this one in some way, but perhaps altered.

Comment (michaelaye) [reply Contributing.md:21]: Oh, gotcha.

File: Charter.md
Comment (rbeyer) [reply Charter.md:58]: The very last item in the document is about the Consensus model.

File: Charter.md
Comment (rbeyer) [reply Charter.md:76]: Um, so the section this text is in is titled 'Collaborators' which is a sibling of a section (before it) called 'Technical Committee Members' and both are subsctions under Membership.  So I feel like the structure of the document pretty much makes it clear that these are two categories, and that they are different.

Comment (michaelaye) [reply Charter.md:76]: well, Andrew seemed to have misunderstood it?

File: Contributing.md
Comment (rbeyer) [reply Contributing.md:43]: It means what it says: 'this repository,' the TC repo.  Again, each repo needs its own `Contributing` document.  I think they could all reference this one, be copied from it, or be logically descended from these contents.

File: Contributing.md
Comment (rbeyer) [reply Contributing.md:73]: If there are two approvals, and for some reason the second approver didn't merge the PR, the PR author (or any committer) could then merge it.  This is identical from the current PSO text.

File: Contributing.md
Comment (rbeyer) [reply Contributing.md:99]: Ii think my answer here is the same as above: this is for this repo, and each repo will have its own document.

File: PSO-Application.md
Comment (rbeyer) [reply PSO-Application.md:40]: I'll admit its a little grammatically artistic.  Suggest a revision.

File: PSO-Application.md
Comment (rbeyer) [reply PSO-Application.md:117]: This PSO-application in our repo is currently a draft.  When we're ready to submit it, we need to make a PR on the PSO TSC repo.  I think the idea is that in this document (unlike our Charter where we do link), the TC member list is explicit so that application review isn't subject to external links.

File: Charter.md
Comment (rbeyer) [reply Charter.md:76]: Well, I think he was just making a general comment, I don't think he was reviewing the document during the meeting.

File: PSO-Application.md
Comment (michaelaye) [reply PSO-Application.md:40]: I was wondering if we even need to evaluate our current status compared to astropy. Isn't it sufficient to pronounce it as our target?

Comment (rbeyer) [reply PSO-Application.md:40]: Good point.  Take a look at the new version, see if you like that better.

**Review by michaelaye — APPROVED** [review]

**Comment by michaelaye** [comment]
I guess we can merge it or do you want a second review?

**Comment by rbeyer** [comment]
We don't currently have a rule in place for two reviews, but the changes I'm suggesting would put that rule in place. Let's see if we can get a second review.

File: Charter.md
Comment (AndrewAnnex) [inline Charter.md:22]: lower case collaborators?

Comment (AndrewAnnex) [reply Charter.md:22]: or not as I see it this way elsewhere..

File: Contributing.md
Comment (AndrewAnnex) [inline Contributing.md]: I think we should say "if they so wish" or something to that effect, there may be situations when significant contributors may not want to participate at a higher level or situations where the TC does not want to approve or invite a contributor.

File: PSO-Application.md
Comment (AndrewAnnex) [inline PSO-Application.md]: why are they not *exactly* the same for the contribution model? Summarize the changes here.

**Review by AndrewAnnex — CHANGES_REQUESTED** [review]
Mostly minor questions, but I do have a question I posted on the "becoming a committer" section that I think deserves some discussion or clarification here. Do we add all significant contributors as committers if they want to be added or not, do they go through the same process of approval (assumed, but not explicitly stated)? Also, direct code contributions aren't a requirement to join as a committer in my view, only interest and good intentions. I think we should state something broader saying that anyone the TC as a whole believes should have committer access should receive it, contributions can serve as positive signals but are not the sole determining factor.

File: Charter.md
Comment (rbeyer) [reply Charter.md:22]: Yeah, using 'capital C' Collaborators signifies that we mean something specific in this document rather than or in addition to the dictionary definition of collaborator.

File: Contributing.md
Comment (rbeyer) [reply Contributing.md]: That is very reasonable.  I guess that this mode of 'automatically' becoming a committer is a simple application of the rules, rather than requiring someone to ask.  This is very equitable, but may end up with us adding a bunch of one-time committers, or those that don't want those powers, as you point out.  Look for a new commit to this PR branch in an hour or so, and I'll update this, see if you like the changed language.

File: PSO-Application.md
Comment (rbeyer) [reply PSO-Application.md]: Also a fine point, look for the next commit to the PR, which will address this.

File: Contributing.md
Comment (michaelaye) [reply Contributing.md]: As a Committer doesn't have obligations but only added rights, i don't understand what the danger is here for someone that doesn't want to participate at a higher level? They aren't forced to?

**Review by michaelaye — APPROVED** [review]

File: Contributing.md
Comment (rbeyer) [reply Contributing.md]: There's no particular 'danger' that I can think of.  From the organization side, there is a potential for a lot of adding of committers that only make a single contribution and then are never seen again, and it just becomes a 'management' issue to keep track of this list of those with commit-access and determine who really needs that access and who doesn't at some later date.

I suspect that in practice, this difference is like splitting hairs, since the TC determines who is made a Collaborator in either case.  It is just in the PSO case, that Collaborator-ship is given solely on the basis of code contribution, and nothing else, and there are also no exceptions.  In this modified case, the TC is allowed to consider other factors when admitting a Collaborator (including the wishes of a particular contributor--if they don't want write-access).  I think this language gives a little more latitude to the TC and the committers.

Comment (michaelaye) [reply Contributing.md]: Good points, thanks.

**Comment by AndrewAnnex** [comment]
Sounds good to me, away from my computer atm but can approve changes later 

-Andrew Annex

**Review by AndrewAnnex — APPROVED** [review]

**Comment by AndrewAnnex** [comment]
@rbeyer should I merge this?

**Comment by rbeyer** [comment]
Yes.

So at this point, anyone could hit the 'merge' button (including me), since it has two approvals.

**Pull request merged**
